packages/webchannel-wrapper/src/index.d.ts

```diff
@@ -31,7 +82,9 @@ export interface WebChannelOptions {
   messageHeaders?: StringMap;
   initMessageHeaders?: StringMap;
```

Comment:
Are these usages of StringMap safe for some reason?  I'm not clear why you had to change `messageUrlParams` and `internalChannelParams` but not these.  

That said, it would be nice if you didn't have to change any of these and our minification logic was able to handle `StringMap`.  Is that not possible?